app/scripts/controllers/attachPVC.js

```diff
@@ -118,6 +118,51 @@ angular.module('openshiftConsole')
         $scope.$watchGroup(['attach.resource', 'attach.allContainers'], updateMountPaths);
         $scope.$watch('attach.containers', updateMountPaths, true);
 
+        var setVolumeMount = function(podTemplate, name, mountPath, subPath, readOnly) {
+          for (var i = 0; i < podTemplate.spec.containers.length; i++) {
```

Comment:
Will switch to an `angular.forEach`.

Reply:
We usually use Lodash `_.each`

https://lodash.com/docs/4.17.4#forEach

Reply:
fwiw, `var` isn't block scoped (inside if statements), switching to `_.each(function() { })` will fix.  It wouldn't error, but its a bit sketch. yay js 😄